styles/styles.css

```diff
@@ -1232,4 +1281,15 @@ main .block.download a::after {
     margin-left: auto;
     max-width: 1200px;
   }
+
+  /* Three Column Layout */
+  main .section.three-columns-layout {
+    height: 1000px;
```

Comment:
We are fixing the height here to 1000px. Wondering if this would work in all cases beyond the current page as well.

Reply:
right, let me fix this.

Reply:
I have introduced mixin "padding-bottom-400" to be more explicit, and it is more clean that the previous height hardcoding.
we can't get rid of this hardcoding completely since we want Three Column Layout section to be positioned according to the specific background image.